src/isomorphic/devtools/ReactInvalidSetStateWarningDevTool.js

```diff
@@ -0,0 +1,19 @@
+var warning = require('warning');
+
+if (__DEV__) {
+  var warnInvalidSetState = function(instance, boolean) {
+    instance._processingChildContext = boolean
+    return;
+  }
+}
+
+var ReactInvalidSetStateWarningDevTool = {
+  onBeginProcessingChildContext(instance, boolean) {
+    warnInvalidSetState(instance, boolean)
```

Comment:
Should set a scope-local variable on the devtool, not warn and/or set variables on the instance.